docs/src/content/docs/protocol/overview.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: "Overview"
+sidebar:
+  order: 1
+---
+
+import Mermaid from "#/components/Mermaid.astro";
+
+The Union protocol is a flavor of IBC (and fully IBC compatible), with certain extensions and innovations to make the protocol more efficient, faster, and flexible for builders, without reducing any security guarantees.
+
+## Connections and Channels
+
+Instead of a legacy single-chain-single-connection protocol, Union uses a connection and channel standard to facilitate communication between rollups and chains. This has several advantages.
```

Comment:
what standard are you referring to here as legacy

Reply:
Older tech like our competitors

Reply:
ah so it's ibc vs <other>, not union vs ibc 👍